### DISCUSSION TITLE
Remove websites from the https://joinmastodon.org/apps

### DISCUSSION THREAD

**Discussion by sysadminpower2019** [mastodon/mastodon#26258] [General]
It seems several of the links provided there especially under the web section are either no longer being support, no longer relevant or have changed completely. One of them doesn't even support social login anymore.
It honestly looks really bad if the official homepage of mastodon isn't properly cared for and maintained.

**Comment by renchap** [comment]
Please open an issue on the website repository: https://github.com/mastodon/joinmastodon/issues

If you can add a list of those website, this could help!

Thanks

**Reply by andypiper** [reply]
agreed - please be more specific about the links that you believe are invalid or unmaintained, in an issue on the joinmastodon repository. Thank you!
